lib/index.js

```diff
@@ -0,0 +1,261 @@
+'use strict';
+
+const path = require('path');
+const querystring = require('querystring');
+const fetch = require('node-fetch');
```

Comment:
We already have `hof-model` in the dependency stack which has all the functionality to do this. See https://github.com/UKHomeOffice/firearms/blob/master/apps/common/models/postcodes.js (EDIT: that's actually a pretty bad example because it doesn't use a bunch of the built-in convenience methods in the model - `auth` etc.)

I'd be wary of adding *another* way of talking to 3rd party APIs to the stack.